docs/resources/vpn_ipsec_gateway.md

```diff
@@ -143,6 +148,10 @@ resource "ionoscloud_vpn_ipsec_gateway" "example" {
       Gateway. **Note**: this should be the subnet already assigned to the LAN
 * `version` - (Required)[string] The IKE version that is permitted for the VPN tunnels. Default: `IKEv2`. Possible
   values: `IKEv2`.
+* `maintenance_window` - (Optional)(Computed)[string] A weekly 4 hour-long window, during which maintenance might occur.
```

Comment:
Not a string, a block or a list. We should fix that for mongo and psql too.

Reply:
@cristiGuranIonos thanks, good catch! I'll modify this.